tests/Benchmarks/System.Buffers/ToSpan.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
```

Comment:
Add the license header

Reply:
Which license file should be added?

Reply:
See other files in the repo. Each file should have the following text at the top of the file:
```csharp
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.
```

For example:
https://github.com/dotnet/corefxlab/blob/ce5c992ecfc1b0dc823a2fd20e4865d2e455b693/src/System.Buffers.Experimental/System/Buffers/Sequences/SequenceExtensions.cs#L1-L4